kevm_pyk/src/kevm_pyk/kevm.py

```diff
@@ -158,3 +86,101 @@ def _patch_symbol_table(symbol_table: Dict[str, Any]) -> None:
         symbol_table['_>=Word__EVM-TYPES_Int_Int_Int']                = paren(lambda a1, a2: '(' + a1 + ') >=Word (' + a2 + ')')            # noqa
         symbol_table['_==Word__EVM-TYPES_Int_Int_Int']                = paren(lambda a1, a2: '(' + a1 + ') ==Word (' + a2 + ')')            # noqa
         symbol_table['_s<Word__EVM-TYPES_Int_Int_Int']                = paren(lambda a1, a2: '(' + a1 + ') s<Word (' + a2 + ')')            # noqa
+
+    @staticmethod
+    def halt() -> KInner:
+        return KApply('#halt_EVM_KItem')
+
+    @staticmethod
+    def execute() -> KInner:
+        return KApply('#execute_EVM_KItem')
+
+    @staticmethod
+    def jumpi() -> KInner:
+        return KApply('JUMPI_EVM_BinStackOp')
+
+    @staticmethod
+    def jump() -> KInner:
+        return KApply('JUMP_EVM_UnStackOp')
+
+    @staticmethod
+    def jumpi_applied(pc: KInner, cond: KInner) -> KInner:
+        return KApply('____EVM_InternalOp_BinStackOp_Int_Int', [KEVM.jumpi(), pc, cond])
+
+    @staticmethod
+    def jump_applied(pc: KInner) -> KInner:
+        return KApply('___EVM_InternalOp_UnStackOp_Int', [KEVM.jump(), pc])
+
+    @staticmethod
+    def pow256():
+        return KApply('pow256_EVM-TYPES_Int', [])
+
+    @staticmethod
+    def range_uint_160(i: KInner) -> KApply:
+        return KApply('#rangeUInt(_,_)_EVM-TYPES_Bool_Int_Int', [intToken(160), i])
+
+    @staticmethod
+    def range_uint_256(i: KInner) -> KApply:
+        return KApply('#rangeUInt(_,_)_EVM-TYPES_Bool_Int_Int', [intToken(256), i])
```

Comment:
```suggestion
    @staticmethod
    def range_uint160(i: KInner) -> KApply:
        return KApply('#rangeUInt(_,_)_EVM-TYPES_Bool_Int_Int', [intToken(160), i])

    @staticmethod
    def range_uint256(i: KInner) -> KApply:
        return KApply('#rangeUInt(_,_)_EVM-TYPES_Bool_Int_Int', [intToken(256), i])
```

uint160/uint256 are how they are called in Solidity.